tests/notebooks/training/training-integration.ipynb

```diff
@@ -415,10 +415,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "PYTORCHJOB_NAME = \"pytorch-dist-mnist-gloo\"\n",
+    "PYTORCHJOB_NAME = \"pytorch-mnist-gloo\"\n",
     "PYTORCHJOB_CONTAINER = \"pytorch\"\n",
-    "PYTORCHJOB_IMAGE = \"kubeflow/pytorch-dist-mnist:v1-3a360ba\"\n",
-    "# The image above should be updated with each release with the latest available in the registry."
+    "PYTORCHJOB_IMAGE = \"kubeflowkatib/pytorch-mnist-cpu:v1beta1-57ed828\"\n",
```

Comment:
where does this specific tag come from?

Reply:
It's the latest available in the registry
># The image above should be updated with each release with the latest available in the registry.

Reply:
after discussing, @orfeas-k noticed the image [has tags](https://hub.docker.com/r/kubeflowkatib/pytorch-mnist-cpu/tags?page=&page_size=&ordering=&name=0.16) for specific katib versions so we decided it's better to use the image that corresponds to the katib version in CKF 1.8 i.e. `0.16.0`